### DISCUSSION TITLE
Modifying the menu, looking for DGUS Tools(DGUS_V5.10)

### DISCUSSION THREAD

**Discussion by Microzod** [Desuuuu/DGUS-reloaded-Klipper#39] [Q&A]
Hello.
I am using your menu(thank you very much) though I very much like to modify it, but I am not able to locate the DGUS Tools.
As far as I can tell the software I am looking for is called DGUS_V5.10 but the company seems to have disappeared and there site is no longer accessible.

So I just wanted to ask, you don't happen to have this software still and able to share it with me?

Regards

**Comment by Desuuuu** [comment]
I know some stuff is available here: https://github.com/juliandroid/DWIN_CR_10s_Pro

Otherwise, most stuff can be found on their chinese forum (dwin.com.cn IIRC).

**Comment by Thinkersbluff** [comment]
Two versions of the DGUSTools are archived on the CR6 Community Firmware GitHub, here:
https://github.com/CR6Community/CR-6-touchscreen/tree/extui/Tools

I recommend you use https://github.com/Tools/DGUS_Tool_V8.2.1.14.7z
